src/org/labkey/test/components/ui/domainproperties/samples/SampleTypeDesigner.java

```diff
@@ -203,5 +209,10 @@ WebElement removeParentAliasIcon(int index)
         {
             return Locator.tagWithClass("i","container--removal-icon").findElements(propertiesPanel).get(index);
         }
+
+        Optional<WebElement> optionalWarningAlert()
+        {
+            return Locator.tagWithClass("div", "alert-warning").findOptionalElement(this);
```

Comment:
There's an existing locator for these banners.
```suggestion
            return BoostrapLocators.warningBanner.findOptionalElement(this);
```

Reply:
awesome, thanks!  I'm looking now to see if putting them in EntityTypeDesigner makes sense